path/dirname.ts

```diff
@@ -20,33 +20,12 @@ import { dirname as windowsDirname } from "./windows/dirname.ts";
  * }
  * ```
  *
- * @param path Path to extract the directory from.
- * @returns The directory path.
- */
-export function dirname(path: string): string;
-/**
- * Return the directory path of a file URL.
- *
- * @experimental **UNSTABLE**: New API, yet to be vetted.
- *
- * @example Usage
- * ```ts
- * import { dirname } from "@std/path/dirname";
- * import { assertEquals } from "@std/assert";
- *
- * if (Deno.build.os === "windows") {
- *   assertEquals(dirname("C:\\home\\user\\Documents\\image.png"), "C:\\home\\user\\Documents");
- *   assertEquals(dirname(new URL("file:///C:/home/user/Documents/image.png")), "C:\\home\\user\\Documents");
- * } else {
- *   assertEquals(dirname("/home/user/Documents/image.png"), "/home/user/Documents");
- *   assertEquals(dirname(new URL("file:///home/user/Documents/image.png")), "/home/user/Documents");
- * }
- * ```
+ * Note: If you are working with file URLs,
+ * use `dirname` from `@std/path/unstable-dirname`.
```

Comment:
This suggests that the unstable API will one day become the stable API. Ditto for the other functions.
```suggestion
 * use the new version of `dirname` from `@std/path/unstable-dirname`.
```